setup.py

```diff
@@ -46,6 +46,7 @@
         "sceptre",
         "sceptre/resolvers",
         "sceptre/hooks"
```

Comment:
Hi @zaro0508  - the build is failing. I think there needs to be a comma after `"sceptre/hooks"` before listing the new package.